gulpfile.js

```diff
@@ -75,14 +79,32 @@ gulp.task('browserify', function() {
     .pipe(gulp.dest(paths.distPublic));
 });
 
-gulp.task('build', ['build-queries', 'copy-js', 'transform-jsx', 'browserify']);
+gulp.task('less', function () {
+  var combined = combiner.obj([
+    gulp.src(paths.lessBundle),
+    less({
+      paths: [
+        'node_modules/bootstrap/less'
+      ]
+    }),
+    gulp.dest(paths.distPublic + '/')
```

Comment:
I have to specify the less file(s) which should result in a final css output file. It's a different thing than making sure that the watch action is triggered when any less file is changed. If I used paths.less, it would result in an output css file for each less file.